modules/json/docs/README.md

```diff
@@ -1,75 +1,37 @@
-# JSONLoader (Experimental)
+# @loaders.gl/json
 
-> This doc is a WIP
+The `@loaders.gl/json` module handles tabular data stored in the [JSON file format](https://www.json.org/json-en.html).
 
-Streaming loader for comma-separated value and [delimiter-separated value](https://en.wikipedia.org/wiki/Delimiter-separated_values) encoded files.
+## Installation
 
-| Loader         | Characteristic                                       |
-| -------------- | ---------------------------------------------------- |
-| File Extension | `.json`,                                             |
-| File Type      | Text                                                 |
-| File Format    | JSON                                                 |
-| Data Format    | [Classic Table](/docs/specifications/category-table) |
-| Supported APIs | `load`, `parse`, `parseSync`, `parseInBatches`       |
-
-## Usage
-
-```js
-import {JSONLoader} from '@loaders.gl/json';
-import {load} from '@loaders.gl/core';
-
-const data = await load(url, JSONLoader, {json: options});
+```bash
+npm install @loaders.gl/core @loaders.gl/json
 ```
 
-The JSONLoader supports streaming JSON parsing, in which case it will yield "batches" of rows of the first array it encounters in the JSON. To e.g. parse a stream of GeoJSON:
-
-```js
-import {JSONLoader} from '@loaders.gl/json';
```

Comment:
Is there other places mentioning how to use this loader?

Reply:
Yes the structure is that the module README just mentions which classes the module provides, and the `api-reference` folder contains a reference for each class/loader.

I am also doing background work on improved user guides for `parseInBatches` and table category.